Tests/MapboxMapsTests/Gestures/GestureHandlers/PinchGestureHandlerTests.swift

```diff
@@ -143,6 +143,69 @@ final class PinchGestureHandlerTests: XCTestCase {
         XCTAssertNil(mapboxMap.setCameraStub.parameters[2].bearing)
     }
 
+    func testPinchChangedWithZoomEnabledFalse() {
+        pinchGestureHandler.zoomEnabled = false
+
+        mapboxMap.cameraState = .random()
+
+        // these touches are consistent with the initial pinch midpoint
+        // and yield an angle of 45° for the initial state
+        gestureRecognizer.getNumberOfTouchesStub.defaultReturnValue = 2
+        gestureRecognizer.getStateStub.defaultReturnValue = .began
+        mapboxMap.dragCameraOptionsStub.defaultReturnValue = CameraOptions(
+            center: .random(),
+            zoom: .random(in: 0...20))
+        gestureRecognizer.sendActions()
+
+        // the new touch angle is 90° - that's 45° increase from the initial.
+        // this should come through as -45° change in bearing since the
+        // coordinate systems are flipped.
+        gestureRecognizer.getStateStub.defaultReturnValue = .changed
+        mapboxMap.dragCameraOptionsStub.defaultReturnValue = CameraOptions(
+            center: .random(),
+            zoom: .random(in: 0...20))
+        gestureRecognizer.sendActions()
+
+        XCTAssertEqual(mapboxMap.setCameraStub.invocations.count, 3)
+        guard mapboxMap.setCameraStub.invocations.count == 3 else {
+            return
+        }
+        XCTAssertNil(mapboxMap.setCameraStub.parameters[0].zoom)
+        XCTAssertNil(mapboxMap.setCameraStub.parameters[2].zoom)
+    }
+
+    func testPinchChangedWithPanEnabledFalse() {
+        pinchGestureHandler.panEnabled = false
+
+        mapboxMap.cameraState = .random()
+
+        // these touches are consistent with the initial pinch midpoint
+        // and yield an angle of 45° for the initial state
+        gestureRecognizer.getNumberOfTouchesStub.defaultReturnValue = 2
+        gestureRecognizer.getStateStub.defaultReturnValue = .began
+        mapboxMap.dragCameraOptionsStub.defaultReturnValue = CameraOptions(
+            center: .random(),
+            zoom: .random(in: 0...20))
+        gestureRecognizer.sendActions()
+
+        // the new touch angle is 90° - that's 45° increase from the initial.
+        // this should come through as -45° change in bearing since the
+        // coordinate systems are flipped.
+        gestureRecognizer.getStateStub.defaultReturnValue = .changed
+        mapboxMap.dragCameraOptionsStub.defaultReturnValue = CameraOptions(
+            center: .random(),
+            zoom: .random(in: 0...20))
+        gestureRecognizer.sendActions()
+
+        XCTAssertEqual(mapboxMap.setCameraStub.invocations.count, 2)
+        guard mapboxMap.setCameraStub.invocations.count == 3 else {
+            return
+        }
+        XCTAssertNil(mapboxMap.setCameraStub.parameters[0].center)
+        XCTAssertNil(mapboxMap.setCameraStub.parameters[2].center)
```

Comment:
Index 2 will be out-of-bounds since we only do 2 setCamera updates in this test.